lib/prompt.rb

```diff
@@ -4,35 +4,63 @@
 require 'board'
 
 class Prompt
-  @board = Board.new
+  attr_accessor :board
 
-  def self.welcome
+  def initialize(board)
+    @board = board
+  end
+
+  def welcome
     print_message(Message.welcome)
   end
 
-  def self.print_instruction
+  def print_instruction
     print_message(Message.instruction)
   end
 
-  def self.print_board
+  def print_board
     print_message(@board.display_board)
   end
 
-  def self.print_current_player(marker)
+  def print_tie
+    print_message(Message.tie)
+  end
+
+  def print_winner(marker)
+    print_message(Message.winner(marker))
+  end
+
+  def print_current_player(marker)
     print_message(Message.current_player(marker))
   end
 
-  def self.print_players_move(marker, move)
+  def print_players_move(marker, move)
     print_message(Message.players_move(marker, move))
   end
 
+  def print_computers_move(marker, move)
+    print_message(Message.computers_move(marker, move))
+  end
+
+  def print_invalid_move_error
+    print_message(Message.invalid_move_error)
+  end
+
   def get_players_move(marker)
     player_move = gets.chomp.to_i
-    self.class.print_players_move(marker, player_move)
+    print_players_move(marker, player_move)
+    sleep 1.5
```

Comment:
What's the purpose of this sleep?

Reply:
I added it to slow the flow of the game down a bit and give time for the players to read the message. I can probably place them in spots where it makes a bit more sense.